build.sbt

```diff
@@ -39,18 +38,25 @@ lazy val chronoscala = (project in file("."))
       )
     },
 
-    libraryDependencies ++= Seq(
-      "org.scalacheck" %% "scalacheck" % "1.14.3" % Test,
-      "org.scalatest" %% "scalatest" % "3.1.1" % Test
-    ),
-
     TaskKey[Unit]("checkScalariform") := {
       val diff = sys.process.Process("git diff").!!
       if (diff.nonEmpty) {
         sys.error("Working directory is dirty!\n" + diff)
       }
     }
   )
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.scalacheck" %%% "scalacheck" % "1.14.3" % Test,
+      "org.scalatest" %%% "scalatest" % "3.1.1" % Test
+    )
+  )
+  .jsSettings(
+    libraryDependencies ++= Seq(
+      "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC4",
```

Comment:
I feel we should wait until GA version has been released

Reply:
from scala-java-time gitter. seems RC4 (it planned to be GA) has some issue, and RC5 has been released soon after that.

![image](https://user-images.githubusercontent.com/6070684/77064707-57f07f00-6a23-11ea-85ef-61e69a1d39ac.png)

Reply:
`2.0.0` Released 🎉 
cf. https://github.com/cquiroz/scala-java-time/releases/tag/v2.0.0